src/scopeHandler.js

```diff
@@ -0,0 +1,13 @@
+const qs = require('querystring')
+
+const scopes = {
+  decode: function (scopesString) {
+    if (Array.isArray(scopesString)) {
+      return scopesString
+    }
+    const decodedScopes = qs.parse(`scope=${scopesString}`)
+    return decodedScopes.scope.split(' ')
+  }
+}
+
+module.exports = scopes
```

Comment:
or just export decode function flat

```js
module.exports = scopesString => {}
```

and rename `scopeHandler.js` to `scopeDecoder`? what do you think? I don't think this file will grow and need a namespace object.